Sources_v3/StreamChatUI/ChatChannelList/ChatChannelView.swift

```diff
@@ -0,0 +1,140 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatChannelView<ExtraData: ExtraDataTypes>: UIView where ExtraData.Channel: NameAndImageProviding {
+    
+    // MARK: - Properties
+    
+    public let uiConfig: UIConfig<ExtraData>
+    
+    public var channel: _ChatChannel<ExtraData>? {
+        didSet { updateAppearance() }
```

Comment:
-> `refreshData()` ??

Reply:
renamed in 6c7a353a11bc8b3283af7b7911fe3f83c7a4b405